src/main/java/testapp/adapter/http/UserService.java

```diff
@@ -0,0 +1,52 @@
+package testapp.adapter.http;
+
+import com.google.inject.Inject;
+import com.google.inject.name.Named;
+import com.google.sitebricks.At;
+import com.google.sitebricks.client.transport.Json;
+import com.google.sitebricks.headless.Reply;
+import com.google.sitebricks.headless.Request;
+import com.google.sitebricks.http.Delete;
+import com.google.sitebricks.http.Get;
+import com.google.sitebricks.http.Post;
+import testapp.core.User;
+import testapp.core.UserAlreadyExistsException;
+import testapp.core.UserDto;
+import testapp.core.UserRepository;
+
+import java.net.HttpURLConnection;
+import java.util.List;
+
+@At("/v1/student")
```

Comment:
Was about to ask you if I should change it to users but I left it as students for that is the way that it will be seen in the browser. Isn't it better for students to see students and not users. I agree that in the backend it will be more readable to be users but in the front end as well ?